packages/backups-cli/src/utils/env.ts

```diff
@@ -100,31 +102,184 @@ export class Environment {
 
     const result = matchingVariables[0].replace(`${fullName}=`, '').trim()
 
-    spinner.succeed(`Successfully obtained key for ${environment} from Vault!`)
+    spinner.succeed(`Successfully obtained keys for ${displayName} from Vault!`)
 
     return result
   }
 
   public static async loadAllFromVault() {
     this.tokens = {
-      [EnvironmentType.LOCAL]: JSON.parse(
-        await this.loadFromVault(
-          EnvironmentType.LOCAL,
-          process.env.ENV_VAR_NAME
-        )
-      ),
-      [EnvironmentType.STAGING]: JSON.parse(
-        await this.loadFromVault(
-          EnvironmentType.STAGING,
-          process.env.ENV_VAR_NAME
-        )
-      ),
-      [EnvironmentType.PRODUCTION]: JSON.parse(
-        await this.loadFromVault(
-          EnvironmentType.PRODUCTION,
-          process.env.ENV_VAR_NAME
-        )
-      ),
+      [EnvironmentType.LOCAL]: {
```

Comment:
The above change will force the update here as well. Let's try using a loop here. If the load env function returns a value it should be stored overwriting current value. If it returns falsy value or error (e.g. lack of permissions) we simply continue.